<?xml version="1.0"?>

<launch>
  <!-- #specifies face recognition algorithm: 0 = Fisherfaces, 1 = Eigenfaces, 2 = LDA2D, 3 = PCA2D, 4 = Deep learning based online recognition
  (attention: choices 0 and 1 work with a single person in the training data, options 2 and 3 will cause a crash of the program if you do not train at least 2 different people)-->
  <arg name="recognition_method" default="2"/>
  <!-- face recognition node (recognizes faces in color image and publishes their positions) -->
  <node name="face_recognizer" pkg="cob_people_detection" type="face_recognizer_node" output="screen">
    <rosparam command="load" file="$(find cob_people_detection)/ros/launch/face_recognizer_params.yaml"/>
    <remap from="~face_positions" to="face_detector/face_positions"/>
    <param name="recognition_method" value="$(arg recognition_method)"/>
  </node>

</launch>
